SoccerPlanner/SoccerPlanner/urls.py

```diff
@@ -43,4 +45,4 @@
     path('stagedeletesuccessful',views.stagedeletesuccessful, name = 'stagedeletesuccessful'),
     path('captcha', views.captcha,name='captcha'),
     path('stageeditsuccessful',views.stageeditsuccessful, name='stageeditsuccessful')
-]
```

Comment:
This should not be removed, since this is an array

Reply:
Ok, done.